csunplugged/at_home/content/en/unlocking-the-secret-in-product-codes/inside-the-computer.md

```diff
@@ -0,0 +1,3 @@
+The digit you calculated is called a checksum, and it is used to make sure that the other digits have been read correctly when the product is scanned or the numbers are typed into a computer. The people who create the barcodes follow this algorithm every time they create a new barcode, and a supermarket checkout computer does the calculation every time it scans in a barcode. The real magic when you scan a product is that if any of the digits are read incorrectly, the check digit will come out wrong, and the computer knows to warn you that the barcode didn’t read in correctly - perhaps the packet is damaged or has some ice covering the barcode. Without the check digit, it would just get the wrong code, and charge you for the wrong product.
```

Comment:
```suggestion
The digit you calculated is called a checksum, and it is used to make sure that the other digits have been read correctly when the product is scanned or the numbers are typed into a computer.
The people who create the barcodes follow this algorithm every time they create a new barcode, and a supermarket checkout computer does the calculation every time it scans in a barcode.
The real magic when you scan a product is that if any of the digits are read incorrectly, the check digit will come out wrong, and the computer knows to warn you that the barcode didn’t read in correctly - perhaps the packet is damaged or has some ice covering the barcode.
Without the check digit, it would just get the wrong code, and charge you for the wrong product.
```